api/envoy/config/filter/udp/dns_filter/v2alpha/dns_filter.proto

```diff
@@ -0,0 +1,64 @@
+syntax = "proto3";
+
+package envoy.config.filter.udp.dns_filter.v2alpha;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.udp.dns_filter.v2alpha";
+option java_outer_classname = "DnsFilterProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.udp.dns_filter.v3";
+option (udpa.annotations.file_status).work_in_progress = true;
+
+// Configuration for the DNS filter.
+message DnsFilterConfig {
+  // Specify the method of determining the address of a given endpoint
+  enum EndpointType { STATIC = 0; }
```

Comment:
Prefer `oneof` for future extensibility.

Reply:
ack.  Will certainly update.